src/chrome/content/rules/VG.no.xml

```diff
@@ -1,22 +1,36 @@
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://vg.no/ => https://vg.no/: Cycle detected - URL already encountered: https://vg.no/
-Fetch error: http://www.vg.no/ => https://www.vg.no/: Cycle detected - URL already encountered: https://www.vg.no/
+	Redirect:
+		- (www.)vg.no
+		- (www.)vgtv.no
+
+	Refused:
+		- direkte.vg.no
 -->
-<ruleset name="VG.no" platform="mixedcontent" default_off='failed ruleset test'>
-	<target host="vg.no" />
-	<target host="www.vg.no" />
+<ruleset name="VG.no">
 	<target host="static.vg.no" />
 	<target host="static01.vg.no" />
 	<target host="static02.vg.no" />
 	<target host="static03.vg.no" />
 	<target host="static04.vg.no" />
-	<rule from="^http://www\.vg\.no/" to="https://www.vg.no/"/>
-	<rule from="^http://vg\.no/" to="https://vg.no/"/>
-	<rule from="^http://static\.vg\.no/" to="https://static.vg.no/"/>
-	<rule from="^http://static01\.vg\.no/" to="https://static01.vg.no/"/>
-	<rule from="^http://static02\.vg\.no/" to="https://static02.vg.no/"/>
-	<rule from="^http://static03\.vg\.no/" to="https://static03.vg.no/"/>
-	<rule from="^http://static04\.vg\.no/" to="https://static04.vg.no/"/>
-</ruleset>
+	<target host="touch.vg.no" />
```

Comment:
This subdomain redirects to ```https://touch.vg.no``` which redirect to ```http://www.vg.no```. This target host basically achieves nothing.